kadalu_operator/main.py

```diff
@@ -567,11 +641,46 @@ def handle_added(core_v1_client, obj):
     update_config_map(core_v1_client, obj)
     deploy_server_pods(obj)
 
+    if wait_till_pod_start() == -1:
+        logging.info(logf("Server pods were not properly deployed"))
+        return
+
     filename = os.path.join(MANIFESTS_DIR, "services.yaml")
     template(filename, namespace=NAMESPACE, volname=volname)
     lib_execute(KUBECTL_CMD, APPLY_CMD, "-f", filename)
     logging.info(logf("Deployed Service", volname=volname, manifest=filename))
 
+    # Time required for service to start
+    time.sleep(20)
+
+    switch = {
+        "Replica1": "replica 1",
+        "Replica3": "replica 3"
+    }
+
+    # TODO: Handle "External" type
+    if obj["spec"]["type"] != "External":
+        vol_type = switch.get(obj["spec"]["type"])
```

Comment:
Handle disperse volume

Reply:
Handled it. Please check if it is right. The disperse volume is being created by this logic.